packages/cozy-stack-client/src/SharingCollection.spec.js

```diff
@@ -31,6 +31,13 @@ describe('SharingCollection', () => {
   const client = new CozyStackClient()
   const collection = new SharingCollection('io.cozy.sharings', client)
 
+  describe('get', () => {
+    it('tests if it calls the right route', async () => {
```

Comment:
```suggestion
    it('should call the right route', async () => {
```

"it" should refer to the thing under test, not to the test :)